transport/src/test/java/io/netty/channel/embedded/EmbeddedChannelTest.java

```diff
@@ -541,6 +542,25 @@ protected void handleOutboundMessage(Object msg) {
         }
     }
 
+    @Test(timeout = 2000)
```

Comment:
Does this have to be this short? Seems like it wants flakiness on a slow machine.